src/torch_models.py

```diff
@@ -337,6 +351,8 @@ def classify(self, x: Array) -> bool:
 
         x is single-dimensional.
         """
+        if self._do_single_class_prediction:
```

Comment:
Let's move this into `_classify` so that the asserts later in this method are still run?

Reply:
I want to bypass the normalization, but I refactored so the asserts are still run.